### ISSUE TITLE
order-by change

### ISSUE THREAD

**Issue by gtod** [orthecreedence/cl-rethinkdb#49]
Hey Andrew, one of the side effects of this recent change (be6dbbbc39561567b8ee0e32ffb30766403b297d) was that the following order-by clause ceases working.  Was that your intention?

```
(re:r (:order-by (:filter (:table "bookings")
                            (re:fn (row)
                              (:&& (:>= (:attr row "time") from-time)
                                   (:<= (:attr row "time") to-time))))
                   "time" "number")))
```

**Comment by orthecreedence** [comment]
No! That was not my intention. Sorry for the problems (I really need to write some tests for this library). My intention was to add the ability to sort by index. Looking at the code, I'm wondering if there's a way to fix this, although it might take an API change, ie going from `&rest fields` to just `fields` and passing in `fields` as a list manually. Either that, or I'm being stupid and I've solved this problem elsewhere without remembering doing so.

I can try to take a look in the next few days, but if you get a chance, please feel free to open a PR.

Thanks for hte heads up!

**Comment by orthecreedence** [comment]
You know, I'm actually wondering if we should just add an `order-by-idx` command. It would preserve backwards compatibility with the old version of the call. Thoughts?

**Comment by gtod** [comment]
Thanks for your help and sorry for my anaemic first post, it was too late at night... `order-by-idx` is fine with me and I feel I should write it for you but I'm too tired to work out if it needs to be `termval` 41 or something else...

But don't do it just on my account either as I now have a quicklisp `local-projects` version of cl-rethinkdb, reverted before this change, and I'll be happy there for sometime/forever...
